dunstrc

```diff
@@ -281,6 +281,8 @@
     #              invoke it. If there are multiple and no default, open the context menu.
     # * close_current: Close current notification.
     # * close_all: Close all notifications.
+    # * context: Open context menu for the notification.
+    # * context_all: Open context menu for all notifications
```

Comment:
Could you also document it in dunst.5.pod?

Reply:
I fixed it in 55dab7e, but noticed a small issue in the makefile: `make clean` does not delete docs/dunst.5, while it does delete the other files for the man pages. Is this worth creating its own PR or should it be fixed in some "small fixes"?

Reply:
Great, could you squash this one as well with the commit where you added the dunstrc documentation.

> make clean does not delete docs/dunst.5, while it does delete the other files for the man pages

Thanks for reporting, I added it to the small fixes PR